custom_components/frigate/media_source.py

```diff
@@ -287,10 +287,6 @@ class RecordingIdentifier(Identifier):
         default=None, validator=[attr.validators.instance_of((str, type(None)))]
     )
 
-    recording_name: str | None = attr.ib(
```

Comment:
This will cause all existing recording identifiers to break. In practice, I think preserving and using these identifiers (e.g. to cast a particular recording) is probably a niche/rare/nonexistent usage.

Reply:
It's been a while since I wrote the media source integration, but I'm pretty certain these identifiers are only used internally. Casting requires a call to play media on a resolved identifier. Since Home Assistant doesn't currently support deep linking into the media browser, there is no reason to save this particular identifier, so we should be in the clear here.